src/pages/App.js

```diff
@@ -43,17 +43,13 @@ const App = ({
               className="text-center lg:text-left text-4xl md:text-6xl font-extrabold
               lg:ml-16 lg:mt-24 w-full lg:w-4/12"
             >
-              {language === 'en' ? (
-                <>
-                  {t('The First African Language ')}
-                  <span className="text-green-500">API</span>
-                </>
-              ) : (
-                <>
-                  <span className="text-green-500">API</span>
-                  {t('The First African Language ')}
-                </>
-              )}
+              {t('The First African Language API').split('$API$')[0]}
```

Comment:
how does this look for English and Igbo? can you show screenshots for this header for these two languages?

Reply:
sure,
[https://i.imgur.com/b9o8omb.png](https://i.imgur.com/b9o8omb.png)